packages/ember-application/lib/ext/controller.js

```diff
@@ -83,7 +83,11 @@ ControllerMixin.reopen({
     For example, when you define a controller:
 
     ```javascript
-    App.CommentsController = Ember.ArrayController.extend({
+    // app/controllers/comments.js
+
+    import Ember from "ember";
+
+    export default Ember.ArrayController.extend({
```

Comment:
@trek Is this type of PR a good idea?

Too soon?

Not formatted properly?

If it's a good idea to move forward with this, I'll dig into other packages but I thought `ember-application` was a good enough package to get started in.